README.md

```diff
@@ -141,6 +156,8 @@ You can proceed by repackaging the project into a zip file and using PhoneGap Bu
 
 For more information about PhoneGap Build, you can refer to their [documentation](http://docs.build.phonegap.com/en_US/index.html).
 
+> Note that a free PhoneGap Build account is limited to a single application, whereas paid plans support multiple applications.
```

Comment:
Clarify a bit more what this implies (eg, need to delete existing app to create new one)